src/components/Calculator/Total/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useCalculator } from "../../../modules/context/Calculator/CalculatorContext";
+
+const Total = () => {
+  const { calculator } = useCalculator();
+
+  return (
+    <h1 id="total">
+      {calculator.total
+        ? calculator.total
+        : calculator.operation && calculator.secondDigits !== "0"
+        ? calculator.secondDigits
+        : calculator.firstDigits}
+    </h1>
+  );
+};
```

Comment:
`calculator` 객체를 한 번 더 구조분해하여 total, operation, secondDigits, firstDigits로 분리한 뒤 중접된 삼항식을 상단에 변수로 선언해두면 어떨까요?

해당 삼항식이 어떤 의도인지, 그 결과값은 무엇인지 제3자로 하여금 유추할 수 있게 분리해주면 가독성이나 직관성이 높아진답니다!

```suggestion
const Total = () => {
  const { calculator: { total, operation, firstDigits, secondDigits  } } = useCalculator();
  const digitValue = operation && secondDigits !== "0"
        ? secondDigits
        : firstDigits
  const total = total ? total : digitValue;

  return (
    <h1 id="total">{total}</h1>
  );
};
```

Reply:
훨씬 더 보기 편해진게 느껴졌습니다!
너무 스스로만 보기 편하게 코드를 짰던 부분이 있던 것 같아 다시 한번 생각하게 됐습니다.